foundation/foundation-mda/src/main/resources/templates/general-docker/mlflow.docker.file.vm

```diff
@@ -7,9 +7,7 @@
 ARG DOCKER_BASELINE_REPO_ID
 ARG VERSION_AISSEMBLE
 
-# NB: will soon be updated to public mlflow image - but waiting to upgrade across the application, so point to the
-# 2.3.1 image that was part of aissemble 1.7.0 in the meantime.  TODO in https://github.com/boozallen/aissemble/issues/226
-FROM ${DOCKER_BASELINE_REPO_ID}boozallen/aissemble-mlflow:1.7.0
+FROM bitnami/mlflow:2.15.1
```

Comment:
Q: Shouldn't this match the tag that's being pulled in the values file?

Reply:
Made the change to match the default tag from the mlflow chart.